lib/tasks/sync_legacy_users.rake

```diff
@@ -0,0 +1,11 @@
+namespace :core do
+  # TODO: Remove once ran on all environments.
+  desc "Creates a LegacyUser object for any existing Users"
+  task sync_legacy_users: :environment do
+    User.where.not(old_user_id: nil).includes(:legacy_users).find_each do |user|
+      next if user.legacy_users.where(old_user_id: user.old_user_id).any?
```

Comment:
I guess this could just be `next if user.legacy_users` since we know any existing users in the system would only have one anyway?

Reply:
I was hoping to make this script idempotent by default so that it could be ran multiple times if needed. The validations later should catch it, but it would raise.